### PR TITLE
ResourceTypes + small tweaks

### PR DESCRIPTION
**Pull request by calebjclark**
- Changed ResourceType to IResourceType. Values are not the same as Chrome Devtools (i.e., no spaces)
- Added ResourceTypes enum export from IResourceType.ts
- Export IResourceType + ResourceTypes enum from hero
- Export ISuperElement and ISuperNode from hero
- Sped up the typing in HumanEmulator.

### CODE REVIEW

File: client/index.ts [hunk 1]
```diff
@@ -30,4 +32,6 @@ export {
   LocationTrigger,
   IHeroCreateOptions,
   IConnectionToCoreOptions,
+  ISuperElement,
```
Comment (blakebyrnes) [inline client/index.ts:35]: Are there other ones of the that should get exported? Like SuperNodeList or anything?

File: interfaces/IResourceType.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Protocol from 'devtools-protocol';
+import Network = Protocol.Network;
+import DevtoolsResourceType = Network.ResourceType;
+
+export enum ResourceTypes {
```
Comment (blakebyrnes) [inline interfaces/IResourceType.ts:6]: Is this supposed to be plural? Seems like most of our enums are singular.

File: core/models/ResourcesTable.ts [hunk 3]
```diff
@@ -304,7 +304,7 @@ export interface IResourcesRecord {
   frameId: number;
   socketId: number;
   protocol: string;
-  type: ResourceType;
+  type: IResourceType;
```
Comment (blakebyrnes) [inline core/models/ResourcesTable.ts:307]: Should this be the enum? Are we just trying to reduce the need for a dev to have to import and use the enum?

File: interfaces/IResourceType.ts (hunk 2 above)
Comment (calebjclark) [reply interfaces/IResourceType.ts:6]: I'm not sure what our standard is here. KeyboardKeys is also exported from @hero/client and it's plural so I followed it.

I can go either way.

File: client/index.ts (hunk 1 above)
Comment (calebjclark) [reply client/index.ts:35]: I don't know, but I needed these two ;)

File: interfaces/IResourceType.ts (hunk 2 above)
Comment (blakebyrnes) [reply interfaces/IResourceType.ts:6]: Hmm. Yeah, that's the only one in the codebase if you search for `enum `

Comment (calebjclark) [reply interfaces/IResourceType.ts]: Fixed to be singular.

**Pull request merged**